aten/src/ATen/native/cuda/Indexing.cu

```diff
@@ -203,7 +203,7 @@ void index_put_accum_kernel(Tensor & self, const c10::List<c10::optional<Tensor>
       using device_ptr = thrust::device_ptr<int64_t>;
       const cudaStream_t stream = at::cuda::getCurrentCUDAStream();
 
-      linearIndex.floor_divide_(sliceSize);
+      linearIndex.divide_(sliceSize, "trunc");
```

Comment:
Are these the only internal floor_divide uses? 

Removing it and trying to compile PyTorch should reveal the remaining .cpp uses.

The Python uses can be detected by throwing a runtime error when the function is called. We should ensure there are no stray uses of a deprecated function, not even in the test suite.

Reply:
>Removing it and trying to compile PyTorch should reveal the remaining .cpp uses.

Removing from `native_functions.yaml` revealed no ther uses. I used `git grep` to find these initially, so should be fairly complete.

>The Python uses can be detected by throwing a runtime error when the function is called.

I found a couple more call sites using this method.

Reply:
Feel free to submit to the Github CI with the function throwing a runtime error so it will run the entire test suite for you to identify floor_divide uses in Python.

Reply:
Submitted to CI and the only non-obvious calls are arising because of the JIT tracer turning `Tensor.size()` into a list of scalar tensors. These should probably be warning though, since they have been silently giving different answers when run through JIT.